### PR TITLE
Update asset specification

### PR DESCRIPTION
**Pull request by venkatrangala**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/HT20-216

This PR is to add additional properties related to Temporary Accommodation in this Shared Assets class library

### CODE REVIEW

File: docs/asset_info.md [hunk 1]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "propertyOccupiedStatus" : "Vacant",
+                     "isNoRepairsMaintenance" : true,
+                     "fundingSource" : "TA",
+                     "costCentre" : "D0720",
+                     "councilTaxType" : "House in multiple occupancy (HMO)",
+                     "councilTaxLiability" : "Void only",
+                     "lhaArea" : "Inner East London",
+                     "isTemporaryAccomodation" : true,
+                     "readyToLetDate" : true,
+                     "assetStatus" : "House",
```
Comment (rashmishetty1011) [inline docs/asset_info.md]: Not sure if this is correct..

Comment (MirelaGeorgiou) [inline docs/asset_info.md]: I believe this field is already covered by 'propertyOccupiedStatus' as mentioned in the spreadsheet.

File: docs/asset_info.md [hunk 2]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "optionToTax" : true
```
Comment (MirelaGeorgiou) [inline docs/asset_info.md]: This is duplicated with the field in the assetCharacteristics

File: docs/asset_info.md [hunk 3]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "fireSafetyCertificateExpiryDate" : "31-12-2030",
+                     "gasSafetyCertificateExpiryDate" : "31-12-2030",
+                     "elecCertificateExpiryDate" : "31-12-2030",
+                     "optionToTax" : true,
+                     "hasStairs" : true,
+                     "numberOfStairs" : 20,
+                     "hasRampAccess" : true,
+                     "hasCommunalAreas" : true,
+                     "hasPrivateBathroom" : true,
+                     "numberOfBathrooms" : 4,
```
Comment (MirelaGeorgiou) [inline docs/asset_info.md:282]: Is a shower room different from a bathroom in terms of the data we store? My concern is introducing fields that we don't store data for / that don't reflect the data collected.

**Review by rashmishetty1011 — CHANGES_REQUESTED** [review]

File: docs/asset_info.md [hunk 4]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "numberOfKitchens" : 4,
+                     "kitchenfloor" : 3,
+                     "alertSystemExpiryDate" : "31-12-2030",
+                     "epcScore" : "80 C",
+                     "numberOfFloors" : 8,
+                     "accessibilityComments" : "There's a step to get into the shower",
+                     "numberOfBedSpaces" : 9,
+                     "numberOfCots" : 12,
+                     "sleepingArrangementNotes" : "Ground Floor",
+                     "numberOfShowers" : 4,
```
Comment (MirelaGeorgiou) [inline docs/asset_info.md:294]: How does that differ to bathrooms/shower room fields above?

Comment (rashmishetty1011) [inline docs/asset_info.md:294]: Repeated here- numberofShowers

**Review by rashmishetty1011 — CHANGES_REQUESTED** [review]

File: docs/asset_info.md [hunk 5]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "councilTaxType" : "House in multiple occupancy (HMO)",
+                     "councilTaxLiability" : "Void only",
+                     "lhaArea" : "Inner East London",
+                     "isTemporaryAccomodation" : true,
+                     "readyToLetDate" : true,
+                     "assetStatus" : "House",
+                     "optionToTax" : true
+              },
+              "assetCharacteristics": {
+                     "numberOfBedRooms": "2",
```
Comment (MirelaGeorgiou) [inline docs/asset_info.md]: 'R' in Bedrooms should be lowercase

File: docs/asset_info.md [hunk 6]
```diff
@@ -184,82 +196,125 @@ Response:
...
+                     "numberOfLifts": "1",
+                     "numberOfLivingRooms": "1",
+                     "windowType": "DBL",
+                     "yearConstructed": "1978",
+                     "numberOfShowerRooms" : 4,
```
Comment (duncanheron) [inline docs/asset_info.md]: not sure this is correct

File: docs/asset_info.md (hunk 3 above)
Comment (venkatrangala) [reply docs/asset_info.md:282]: I think the confusion was caused by the name numberOfShowerRooms, this should have been numberOfShowers which is a metric we need to record as not every bathroom would have a shower.

File: docs/asset_info.md (hunk 2 above)
Comment (venkatrangala) [reply docs/asset_info.md]: Removed

File: docs/asset_info.md (hunk 1 above)
Comment (venkatrangala) [reply docs/asset_info.md]: Agreed, removed this assetStatus field now

Comment (venkatrangala) [reply docs/asset_info.md]: We have removed this field now

File: docs/asset_info.md (hunk 4 above)
Comment (venkatrangala) [reply docs/asset_info.md:294]: I think the confusion was caused by the name numberOfShowerRooms, this should have been numberOfShowers which is a metric we need to record as not every bathroom would have a shower

Comment (venkatrangala) [reply docs/asset_info.md:294]: This is the correct one, have removed numberofShowerRooms

File: docs/asset_info.md (hunk 5 above)
Comment (venkatrangala) [reply docs/asset_info.md]: fixed the casing

File: docs/asset_info.md (hunk 6 above)
Comment (venkatrangala) [reply docs/asset_info.md]: Have removed this as numberofShowers is the correct one

**Review by MirelaGeorgiou — APPROVED** [review]

**Review by rashmishetty1011 — APPROVED** [review]

**Pull request merged**